packages/main/src/themes/Input.css

```diff
@@ -83,6 +84,7 @@
 [inner-input]::-webkit-input-placeholder {
 	font-style: italic;
 	color: var(--sapField_PlaceholderTextColor);
+	padding-right: 2px;
```

Comment:
This affects placeholder, and the rare case (in general) when the alignment of the input is "right", then, because the placeholder uses italic font, 2 pixels of the text are cut... this padding fixes the issue.